core/src/stages/compute_ik.cpp

```diff
@@ -314,6 +314,11 @@ void ComputeIK::compute() {
 
 		link = scene->getCurrentState().getRigidlyConnectedParentLinkModel(ik_pose_msg.header.frame_id);
 
+		if (!link) {
```

Comment:
`knowsFrameTransform()`, a few lines above, should have validated that the frame exists.
Under what circumstances do you run into a segfault here?
I cannot imagine a valid frame that returns NULL here.

Reply:
There seem to be cases under which [getRigidlyConnectedParentLinkModel](https://github.com/moveit/moveit2/blob/aece55c3ebe4a88bb9c09b85fa12479b94690c0b/moveit_core/robot_state/src/robot_state.cpp#L910) returns a nullptr. Are they all covered by `knowsFrameTransform`?

Reply:
> There seem to be cases under which [getRigidlyConnectedParentLinkModel](https://github.com/moveit/moveit2/blob/aece55c3ebe4a88bb9c09b85fa12479b94690c0b/moveit_core/robot_state/src/robot_state.cpp#L910) returns a nullptr. Are they all covered by `knowsFrameTransform`?

Yes, the should be covered by `knowsFrameTransform`. If not, that's probably a bug. Please state your concrete problem.